src/components/assign-lesson/compile.directive.js

```diff
@@ -0,0 +1,27 @@
+(function (angular) {
+    'use strict';
+
+    var module = angular.module('znk.infra-dashboard.assign-lesson');
+
+    module.directive('compile', ['$compile', function ($compile) {
+        return {
```

Comment:
we already have in infra 
angular.module('znk.infra.general').directive('compile'

do we need this one also?